reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -380,23 +380,24 @@ class StaticHtmlReporter : Reporter() {
 
         ul {
             reportTableModel.evaluatorIssues?.let { ruleViolations ->
-                val issues = ruleViolations.groupBy { it.severity }
+                val issues = ruleViolations.filterNot { it.isResolved }.groupBy { it.severity }
                 val errorCount = issues[Severity.ERROR].orEmpty().count()
                 val warningCount = issues[Severity.WARNING].orEmpty().count()
                 val hintCount = issues[Severity.HINT].orEmpty().count()
 
                 li {
                     a("#rule-violation-summary") {
-                        +"Rule Violation Summary ($errorCount errors, $warningCount warnings, $hintCount hints)"
+                        +("Rule Violation Summary ($errorCount errors, $warningCount warnings, $hintCount hints to " +
+                                "resolve)")
                     }
                 }
             }
 
-            if (reportTableModel.issueSummary.allIssues.isNotEmpty()) {
+            if (reportTableModel.issueSummary.unresolvedIssues.isNotEmpty()) {
```

Comment:
This neeeds to check for `rows.isNotEmpty()` now, otherwise the table disappears if all issues are resolved.

Reply:
Thanks, done.